src/queue.js

```diff
@@ -212,4 +215,16 @@ Queue.prototype._dequeuePendingRequest = function(item) {
   }
 };
 
+Queue.prototype._maybeLog = function(item) {
+  if (this.logger && this.options.verbose) {
+    var message = _.get(item, 'data.body.trace.exception.message');
```

Comment:
Gotcha. If it's easy, I'd like to add verbose logging for messages as well as exceptions. Basically, anytime someone calls one of the top-level API methods which result in a POST.